### PR TITLE
Added benchmark test for listener.Flush()

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What this PR does / why we need it**:
Added benchmark test for listener.Flush()

### CODE REVIEW

File: pkg/cache/v2/listener_test.go [hunk 1]
```diff
@@ -180,3 +184,93 @@ func TestListenerFlush(t *testing.T) {
 		assert.Equal(t, []*core_v2.SocketAddress{}, deleteListenerAddress)
 	})
 }
+
+func BenchmarkFlush(b *testing.B) {
+	t := &testing.T{}
+	test.InitBpfMap(t)
+	t.Cleanup(test.CleanupBpfMap)
```
Comment (hzxuzhonghu) [inline pkg/cache/v2/listener_test.go:191]: can the cleanup be called?

B.CleanUp can be used here

File: pkg/cache/v2/listener_test.go [hunk 2]
```diff
@@ -180,3 +184,93 @@ func TestListenerFlush(t *testing.T) {
...
+		},
+	}
+
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		cache := NewListenerCache()
+		listener.Name = rand.String(6)
+		cache.SetApiListener(listener.Name, listener)
+
+		cache.Flush()
```
Comment (hzxuzhonghu) [inline pkg/cache/v2/listener_test.go:273]: Do you think it really can call bpf update?

File: pkg/cache/v2/listener_test.go [hunk 3]
```diff
@@ -180,3 +184,93 @@ func TestListenerFlush(t *testing.T) {
...
+						Name: "filter2",
+						ConfigType: &listener_v2.Filter_TcpProxy{
+							TcpProxy: &filter.TcpProxy{
+								StatPrefix: "outbound|80||kube-dns.kube-system.svc.cluster.local",
+								ClusterSpecifier: &filter.TcpProxy_Cluster{
+									Cluster: "outbound|80||kube-dns.kube-system.svc.cluster.local",
+								},
+								MaxConnectAttempts: uint32(60),
+							},
+						},
```
Comment (hzxuzhonghu) [inline pkg/cache/v2/listener_test.go:224]: This can not exist in real case, usually one filterChain can point to one route/one cluster.

**Comment by LiZhenCheng9527** [comment]
```console
goos: linux
goarch: amd64
pkg: kmesh.net/kmesh/pkg/cache/v2
cpu: Intel(R) Xeon(R) Gold 6278C CPU @ 2.60GHz
BenchmarkFlush-4             100        4974271585 ns/op
PASS
ok      kmesh.net/kmesh/pkg/cache/v2    497.555s
```

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**